pygrametl/tables.py

```diff
@@ -1863,6 +1866,14 @@ def __init__(self, name, keyrefs, measures=(), batchsize=10000,
 
         self.__batchsize = batchsize
         self.__batch = []
+        if usevalues:
+            self.__insertnow = self.__insertvalues
+            self.__basesql = self.insertsql[:self.insertsql.find(' (') + 1]
+            self.__rowtovalue = lambda row: '(' + \
+                ','.join(map(lambda c: "'" + row[c] + "'" if type(row[c])
+                             is str else str(row[c]), self.all)) + ')'
```

Comment:
Couldn't we escape backslashes and single quotes in `row[c]` when it is a string? In other words, use `row[c].replace("\\", "\\\\").replace("'", "\\'")`?